concent_api/concent_api/tests/test_check_siginig_service_key_availability_for_middleman.py

```diff
@@ -17,9 +18,11 @@ def test_that_if_middleman_is_disabled_signing_service_public_key_does_not_have_
             assert_that(errors).is_empty()
 
     def test_that_if_middleman_is_enabled_and_key_signing_service_public_key_is_not_defined_error_is_returned(self):
-        errors = check_signing_service_key_availability_for_middleman()
-        assert_that(errors).is_length(1)
-        assert_that(errors[0]).is_equal_to(create_error_43_signing_service_public_key_is_missing())
+        with override_settings():
```

Comment:
this change doesn't seem to belong to this PR ;)

Reply:
True, actually. I don't know why I've added that `override_settings()`. `del settings` works just fine without it.